### DISCUSSION TITLE
How to suggest new quotes?

### DISCUSSION THREAD

**Discussion by vpodolyan** [lukePeavey/quotable#68] [Questions] [answered]
Hi @lukePeavey!
I'm using Quotable API in my pet project and would like to add some quotes to the production env. Is there a way to do that?
I could just send you a list of quotes I'd like to add

**Comment by lukePeavey** [comment, marked as answer]
I would be happy to add your quotes to the database! At some point, I am hoping to set up a way for people to contribute directly to the collection. 

But for now, I created a discussion topic for New Quote Suggestions #69.

**Reply by vpodolyan** [reply]
Great, thank you for quick reply! I'll post something there soon  :slightly_smiling_face:

**Reply by reivenIV** [reply]
Hi i m a newby in the coding field (started since 1 month).. and i waas looking a free API opensource and i end up here :) ... I got the same question as Vladimir.. I was thinking to generate a quote app with spiritual quotes.. I wanted to bring all these amazing quotes to quotable.. but i see that is not possible... 

So my question is.. do you have any tutorial to learn how to create an API..? is it hard? 

And it would be interesting to know why is not open to anyone tu add quotes freely ? because i was thinking to do so.. 
Thanks a lot. I m new to API's and i found it super interesting. 

Thanks a lot. 
Federico

**Reply by lukePeavey** [reply]
@Otarwebe The API data is open source, its managed in separate repository.   If you would like to submit new quotes, you are welcome to contribute. 

Here is the link to the data repository
https://github.com/quotable-io/data

**Reply by reivenIV** [reply]
Thanks for the quick reply ! I will check that out !
